dockers/docker-database/database_config.json.j2

```diff
@@ -72,6 +72,11 @@
             "id" : 11,
             "separator": "|",
             "instance" : "redis"
+        },
+        "GLOBAL_STATE_DB" : {
+            "id" : 12,
+            "separator": "|",
+            "instance" : "redis-global_db"
```

Comment:
same comment as on line 76.

Reply:
I think you won't need this PR after rebase.

Reply:
We are adding a CHASSIS_STATE_DB so that the pmon state is kept separate from the data-path state in CHASSIS_APP_DB. This is inline with what happens on local redis instance.